PhoneBook.Doc415/DataAccess.cs

```diff
@@ -0,0 +1,52 @@
+﻿using PhoneBook.Doc415.Context;
+using PhoneBook.Doc415.Models;
+
+namespace PhoneBook.Doc415;
+
+internal class DataAccess
+{
+    public void AddContact(string _name, string _email, string _phone, string _title)
```

Comment:
`_variableName` it's used for private variables, not for arguments of functions. 
Although if that's what you prefer, it should not matter.

Reply:
What you say is true for python. If it is a tradition in c# i am not aware of it. It reminds me that i am working with an object out of scope. A habit from "processing" language